pyrasterframes/src/main/python/pyrasterframes/rasterfunctions.py

```diff
@@ -348,6 +362,16 @@ def rf_render_matrix(tile_col):
     return _apply_column_function('rf_render_matrix', tile_col)
 
 
+def rf_render_png(red_tile_col, green_tile_col, blue_tile_col):
```

Comment:
we should have a single tile overload or another function to do the ColorRamp

Reply:
#313